core/vm/interpreter.go

```diff
@@ -55,7 +55,10 @@ type EVMInterpreter struct {
 func NewEVMInterpreter(evm *EVM) *EVMInterpreter {
 	// If jump table was not initialised we set the default one.
 	var table *JumpTable
+	customTable := evm.chainRules.Hooks().JumpTable()
 	switch {
+	case customTable != nil:
+		table = customTable.(*JumpTable)
```

Comment:
See this comment for discussion: https://github.com/ava-labs/go-ethereum/pull/11#discussion_r1757607538